fsi/solutions/bankingCustomerEngagement/readme.md

```diff
@@ -29,12 +29,12 @@ Specifically for the banking customer engagement scenario, you need:
     * Omnichannel for Customer Service
     * Customer Insights (optional but required for full experience)
 
-> Note: The implementation of the Banking Customer Engagement solution can currently not be automated, and requires manual implementation and configuration across Solution Center, Power Platform Admin Center.
+> Note: The implementation of the Banking Customer Engagement solution can currently not be automated, and requires manual implementation and configuration across Solution Center and Power Platform Admin Center.
```

Comment:
Consider changing "... requires manual implementation" to either ".. requires manual inputs across Solution Center and Power ..."

?

Reply:
This is currently the consistent wording for all docs. I would prefer to keep it consistent here as well. What do you think?